test/parallel/test-fs-promises-file-handle-readFile.js

```diff
@@ -35,6 +36,29 @@ async function validateReadFile() {
   await fileHandle.close();
 }
 
+async function validateLargeFileSupport() {
+  const LARGE_FILE_SIZE = 3 * 1024 * 1024 * 1024; // 3 GiB
```

Comment:
Allocating such big files takes a long time on many machines and that makes it difficult for developers to contribute.

Please use a limit just slightly above our current maximum, since it would already reflect that it's possible to go beyond INT32. That applies to all of these tests :)

Reply:
thanks, I edited